docs/basics/ui-patterns.md

```diff
@@ -0,0 +1,192 @@
+---
+title: UI-Patterns
+order: 6
+---
+
+# UI Patterns
```

Comment:
```suggestion
# UI Patterns

> This page is incomplete. You can help us expanding it by suggesting more patterns or asking us about common problems you're facing on [GitHub Discussions](https://github.com/FormidableLabs/urql/discussions).
```

I thought it'd be neat to just ask people for help here, since it won't cost anything and is the perfect place to do so.